docs/components/navbar.md

```diff
@@ -178,6 +178,14 @@ Navbars can be statically placed (their default behavior), or fixed to the top o
 </nav>
 {% endexample %}
 
+When using full width, statically placed navbars you might want to remove the rounded corners, this can be done by including `.navbar-static-top`.
```

Comment:
Is there any reason why this component should even have the `-top` extension? Like @tabone says, there's nothing to denote the position of the component?